packages/presentation/lib/screen/cast_list/cast_crew_screen.dart

```diff
@@ -0,0 +1,133 @@
+import 'package:domain/model/people_and_images_model.dart';
+import 'package:flutter/material.dart';
+import 'package:presentation/base/bloc_data.dart';
+import 'package:presentation/base/bloc_screen.dart';
+import 'package:presentation/config/dimens/dimens.dart';
+import 'package:presentation/config/responsive/responsive.dart';
+import 'package:presentation/config/text_style/text_style.dart';
+import 'package:presentation/config/theme/app_colors.dart';
+import 'package:presentation/generated_localization/l10n.dart';
+import 'package:presentation/navigation/base_arguments.dart';
+import 'package:presentation/navigation/base_page.dart';
+import 'package:presentation/screen/cast_list/cast_crew_bloc.dart';
+import 'package:presentation/screen/cast_list/cast_crew_data.dart';
+import 'package:presentation/screen/movie_details/widgets/details_reviews_shimmer.dart';
+import 'package:presentation/screen/movie_details/widgets/movie_list_actors.dart';
+
+class CastCrewScreenArguments extends BaseArguments {
+  CastCrewScreenArguments({
+    required this.detailsAboutPeople,
+    super.result,
+  });
+
+  final List<PeopleAndImagesModel>? detailsAboutPeople;
+}
+
+class CastCrewScreen extends StatefulWidget {
+  const CastCrewScreen({super.key});
+
+  static const _routeName = '/CastListScreen';
+
+  static BasePage page(CastCrewScreenArguments arguments) => BasePage(
+        key: const ValueKey(_routeName),
+        name: _routeName,
+        builder: (context) => const CastCrewScreen(),
+        showSlideAnim: true,
+        arguments: arguments,
+        isButtonNavBarActive: false,
+      );
+
+  @override
+  State<CastCrewScreen> createState() => _CastCrewScreenState();
+}
+
+class _CastCrewScreenState
+    extends BlocScreenState<CastCrewScreen, CastCrewBloc> {
+  @override
+  Widget build(BuildContext context) {
+    final width = MediaQuery.of(context).size.width;
+    return StreamBuilder<BlocData<CastCrewData?>>(
+        stream: bloc.dataStream,
+        builder: (context, snapshot) {
+          final data = snapshot.data;
+          final CastCrewData? blocData = data?.data;
+          final details = blocData?.detailsAboutPeople;
+          final castLength = details?.length;
```

Comment:
you either specify the type of the variable everywhere, or do not specify it, since flutter will substitute it

Reply:
okay